src/components/wishlistCard/wishListCard.css

```diff
@@ -0,0 +1,16 @@
+.MuiContainer-root.removed-item{
+  animation: removed-item-animation .6s cubic-bezier(.55,-0.04,.91,.94) forwards;
+}
+
+@keyframes removed-item-animation {
+  from {
+      opacity: 1;
+      transform: scale(1);
+  }
+
+  to {
+      opacity: 0;
+      height: 0px;
+      transform: scale(0);
+  }
+}
```

Comment:
this one

Reply:
Oh yes forgot about that file. wishListCard component is not being used anywhere right now.